### ISSUE TITLE
New ignite project with expo error on Android QR code scan

### ISSUE THREAD

**Issue by gwguillen** [infinitered/ignite#1917]
What's going on?
New project with ignite-cli for expo errors on Android QR with "Cannot delete property 'resolveRequest' of Object.
This does not occur with a new expo-cli project.
These unrecognized events occur regardless. The TypeError occurs upon scanning the QR code with Android device.
---

Steps to reproduce

1. npx ignite-cli new IgniteApp --expo
2. npm run start
3. Scan QR code with Android device
---
Unrecognized event: {"type":"transformer_load_started"} 
Unrecognized event: {"type":"transformer_load_done"}
Started Metro Bundler
TypeError: Cannot delete property 'resolveRequest' of #<Object> 
    at C:\Code\IgniteApp\node_modules\@rnx-kit\metro-resolver-symlinks\lib\index.js:13:9
    at Object.resolve (C:\Code\IgniteApp\node_modules\metro-resolver\src\resolve.js:31:12)
    at ModuleResolver.resolveDependency (C:\Code\IgniteApp\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:115:31)
    at DependencyGraph.resolveDependency (C:\Code\IgniteApp\node_modules\metro\src\node-haste\DependencyGraph.js:231:43)
    at C:\Code\IgniteApp\node_modules\metro\src\lib\transformHelpers.js:129:24
    at Server._resolveRelativePath (C:\Code\IgniteApp\node_modules\metro\src\Server.js:1107:12)
    at Server.requestProcessor [as _processBundleRequest] (C:\Code\IgniteApp\node_modules\metro\src\Server.js:450:37)
    at Server._processRequest (C:\Code\IgniteApp\node_modules\metro\src\Server.js:406:9)
---

`npx ignite-cli doctor` results:
System
  platform           win32
  arch               x64
  cpu                8 cores      Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
  directory          IgniteApp    C:\Code\IgniteApp

JavaScript (and globally-installed packages)
  node               16.14.0      C:\Program Files\nodejs\node.EXE
  npm                8.5.3        C:\Users\USER\AppData\Roaming\npm\npm.CMD
    @expo/ngrok      4.1.0
    expo-cli         5.2.0
    npm              8.5.3
  yarn               -            not installed
  pnpm               -            not installed

Ignite
  ignite-cli         7.10.6       C:\Users\USER\AppData\Local\npm-cache\_npx\e31027f3785124a8\node_modules\.bin\ignite.CMD
  ignite src         build        C:\Users\USER\AppData\Local\npm-cache\_npx\e31027f3785124a8\node_modules\ignite-cli\build

Android
  java               1.8.0_201    C:\Program Files (x86)\Common Files\Oracle\Java\javapath\java.EXE
  android home       -            C:\Users\USER\AppData\Local\Android\Sdk

**Comment by aqjw** [comment]
any solutions?

**Comment by everbliss7** [comment]
Getting the same error.

**Comment by everbliss7** [comment]
@aqjw can you let us know why you closed this issue, please?

**Comment by aqjw** [comment]
@everbliss7 Hi!
it's not me who closed this issue, but I had the same problem.

The problem is in one of the packages you installed recently!
In my case, the problem was because I installed two packages that did the same thing.
`react-native-date-picker` and `@react-native-community/datetimepicker`.
after removing `react-native-date-picker` everything worked.

Try rolling back a few commits, then do:
1. remove the node_models folder and run `npm install`
2.  run `expo install`

I hope I helped you!
